src/main/java/nl/haarlem/translations/zdstozgw/converter/impl/replicate/Replicator.java

```diff
@@ -195,11 +195,20 @@ private void checkVoegZaakDocumentToe(String zaakidentificatie, String rsin, Lis
             		this.converter.getSession().setAantalDocumentenGerepliceerd(adg + 1);
             	}
             	catch(ConverterException ex) {
-            		debug.infopoint("converter exception", "document with identificatie #" + zaakdocumentidentificatie + " has error" + ex.toString());
-            		// ignore some errors, TODO: make it configurable
-
+            		debug.infopoint("converter exception", "document with identificatie #" + zaakdocumentidentificatie + " has error:" + ex.toString());
+            		if(ex.details != null) {
+            			debug.infopoint("converter exception", "document with identificatie #" + zaakdocumentidentificatie + " has error:" + ex.toString());
+            		}           		
+            		if(ex.details != null) {
+            			debug.infopoint("converter exception-details", ex.details);
```

Comment:
this can be in the if statement above. Now it is duplication the same check ex.details != null